Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -54,9 +54,16 @@ def PyInit(self):
                              direction=Direction.Output),
                              ("Name the workspace that will contain the " +
                               "calibration coefficients"))
+        self.declareProperty("Temperature",
+                             defaultValue=Property.EMPTY_DBL,
+                             validator=FloatBoundedValidator(lower=0.0),
+                             direction=Direction.Input,
+                             doc=("Temperature during the experiment."))
         return
 
     def validateInputs(self):
+        """ Validate the inputs
+        """
         issues = dict()
         inws = self.getProperty("VanadiumWorkspace").value
         run = inws.getRun()
```

Comment:
@NickDraper It is very interesting what you have observed. Theoretically, it should not happen. The `VanadiumWorkspace` property is validated in `PyInit`:

```python
self.declareProperty(MatrixWorkspaceProperty(
                             "VanadiumWorkspace", "",
                             direction=Direction.Input,
                             validator=InstrumentValidator()),
                             "Input Vanadium workspace")
```
If one runs
```python
result = ComputeCalibrationCoefVan(VanadiumWorkspace=None, EPPTable='wsEppTable')
```
one should get the `RuntimeError: Some invalid Properties found` and the `validateInputs()` will not be executed.

Please, try to find the way to reproduce the error you have observed. It is really very interesting. I did not manage to reproduce it neither on Mac nor on Win7.